### PR TITLE
(DOCSP-17523,DOCSP-17766): TCK parameter in mongodb.com links, atlas form container links

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
Two small things to check:
- many of the existing links used the tck parameter `realm-docs`, but DOCSP-17523 claims the parameter is `docs_realm` (note the underscore and reversed order). Which is actually correct?
- Based on my knowledge of query parameters, I feel like the link
  [https://www.mongodb.com/realm?tck=docs_realm#atlas-form-container](https://www.mongodb.com/realm?tck=docs_realm#atlas-form-container) should actually be
  [https://www.mongodb.com/realm#atlas-form-container?tck=realm-docs](https://www.mongodb.com/realm#atlas-form-container?tck=realm-docs), because query parameters typically come last after the entire url. But If I try that link instead, I don't get an atlas form. So maybe I just forgot where query parameters go relative to `#` arguments in URLs.

### Jira

- https://jira.mongodb.org/browse/DOCSP-17523

### Staged Changes (Requires MongoDB Corp SSO)

- [Staging build](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-17523/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/tutorial/realm-app.txt [hunk 1]
```diff
@@ -43,7 +43,7 @@ an existing MongoDB Atlas Account, you can
 an Atlas account, follow the steps below to create one:
 
 1. Navigate to the `MongoDB Atlas login page
-<https://account.mongodb.com/account/register>`__.
+<https://account.mongodb.com/account/register?tck=docs_realm>`__.
```
Comment (nathan-contino-mongo) [inline source/tutorial/realm-app.txt:46]: The ticket mentions that we don't actually need this here yet, but it came up in the SEO report and it doesn't hurt, so I figure we might as well get ahead of it.

**Review by dacharyc — APPROVED** [review]
LGTM with a couple of Qs:

The ticket says "all links to mongodb.com" should have the tck - should this include these links?
- [Get Help ln 27](https://github.com/mongodb/docs-realm/blob/11a4b746474e6cf75264f7815944590538aac998/source/get-started/getting-help.txt#L27)
- [Get Help ln 41](https://github.com/mongodb/docs-realm/blob/11a4b746474e6cf75264f7815944590538aac998/source/get-started/getting-help.txt#L41)
- A couple of other places the feedback.mongodb.com link appears
- Some links on the .Net Remote Access page: [ln 253](https://github.com/mongodb/docs-realm/blob/11a4b746474e6cf75264f7815944590538aac998/source/sdk/dotnet/examples/mongodb-remote-access.txt#L253), [ln 269](https://github.com/mongodb/docs-realm/blob/11a4b746474e6cf75264f7815944590538aac998/source/sdk/dotnet/examples/mongodb-remote-access.txt#L269), [ln 288](https://github.com/mongodb/docs-realm/blob/11a4b746474e6cf75264f7815944590538aac998/source/sdk/dotnet/examples/mongodb-remote-access.txt#L288)
- A link on the [Sync Errors page](https://github.com/mongodb/docs-realm/blob/11a4b746474e6cf75264f7815944590538aac998/source/sync/errors.txt#L24)
- A handful of links to developer.mongodb.com

**Comment by nathan-contino-mongo** [comment]
Just `mongodb.com`, not `docs.mongodb.com`, so I think we're good!

**Comment by sumedhamehta** [comment]
not sure about the right tck here (maybe docs platform is more familiar with this) , but the right URL to go to is discussed here: https://mongodb.slack.com/archives/CJ9105ZSA/p1616518295014100 (also tagged you nate)

**Pull request merged**